_utility/send_gh_comment_validation.sh

```diff
@@ -0,0 +1,105 @@
+# Define help function
```

Comment:
A note on how did you generate this and how a next maintainer might re-generate with minimal effort might be helpful here